web/interactive_terminal/jupyter-lite.json

```diff
@@ -0,0 +1,11 @@
+{
+  "jupyter-lite-schema-version": 0,
+  "jupyter-config-data": {
+    "appName": "Pandas REPL",
+    "appUrl": "./repl",
+    "disabledExtensions": ["@jupyter-widgets/jupyterlab-manager"],
```

Comment:
Do we want to disable support for widgets in the REPL?

Otherwise it sounds like being able to use ipywidgets could still be useful, even though it's not strictly related to pandas.

Reply:
Ah, I kept it the same as how your previous PR was back then, but I'm happy to remove it!

Addressed in 59f3e1619442ae3b03bb683abc6a0611b8081181

Reply:
Ah right, it's probably because widgets were not supported yet at the time, and there were likely a few errors in the dev tools console then.